cpp/src/arrow/compute/kernels/CMakeLists.txt

```diff
@@ -73,6 +73,7 @@ add_arrow_compute_test(scalar_utility_test
                        scalar_random_test.cc
                        scalar_set_lookup_test.cc
                        scalar_validity_test.cc
+                       scalar_hash_test.cc
```

Comment:
```suggestion
                       scalar_hash_test.cc
                       scalar_random_test.cc
                       scalar_set_lookup_test.cc
                       scalar_validity_test.cc
```

Reply:
Might want to keep source list in alphabetical.

Reply:
Fixed.